.github/workflows/ci-sharktank-nightly.yml

```diff
@@ -7,10 +7,10 @@
 name: CI - sharktank nightly
 
 on:
-  workflow_dispatch:
-  schedule:
-    # Weekdays at 11:00 AM UTC = 03:00 AM PST / 04:00 AM PDT
-    - cron: "0 11 * * 1-5"
+  push:
+    branches:
+      - 'fix_flux_ci'
```

Comment:
This would need to be reverted.
You can trigger manually a workflow from the [actions panel](https://github.com/nod-ai/shark-ai/actions). There you can select a branch that is in this repo.